diff -ruN ./src/slic3r/GUI/Field.cpp ../../source/src/slic3r/GUI/Field.cpp
--- ./src/slic3r/GUI/Field.cpp	1970-01-01 01:00:01.000000000 +0100
+++ ../../source/src/slic3r/GUI/Field.cpp	2023-07-20 01:19:02.755757640 +0200
@@ -1539,8 +1539,9 @@
 
     wxButton* btn = dynamic_cast<wxButton*>(field->GetPickerCtrl());
     wxImage image(btn->GetBitmap().GetSize());
-    image.InitAlpha();
-    memset(image.GetAlpha(), 0, image.GetWidth() * image.GetHeight());
+    // TODO: why initAlpha() raise segv?
+    /*image.InitAlpha();
+    memset(image.GetAlpha(), 0, image.GetWidth() * image.GetHeight());*/
     wxBitmap   bmp(std::move(image));
     wxMemoryDC dc(bmp);
     if (!dc.IsOk()) return;
