diff -ruN ./src/src/slic3r/GUI/Field.cpp ./source/src/slic3r/GUI/Field.cpp
--- b/src/src/slic3r/GUI/Field.cpp	2023-10-24 17:23:31.535256565 +0200
+++ a/src/slic3r/GUI/Field.cpp	2023-10-24 17:24:37.187805397 +0200
@@ -1540,8 +1540,8 @@
 
     wxButton* btn = dynamic_cast<wxButton*>(field->GetPickerCtrl());
     wxImage image(btn->GetBitmap().GetSize());
-    image.InitAlpha();
-    memset(image.GetAlpha(), 0, image.GetWidth() * image.GetHeight());
+    /*image.InitAlpha(); TODO: SEGV fix-me
+    memset(image.GetAlpha(), 0, image.GetWidth() * image.GetHeight());*/
     wxBitmap   bmp(std::move(image));
     wxMemoryDC dc(bmp);
     if (!dc.IsOk()) return;
