diff -ruN ./src/CMakeLists.txt ../../source/src/CMakeLists.txt
--- ./src/CMakeLists.txt	2023-07-25 20:29:47.954439000 +0200
+++ ../../source/src/CMakeLists.txt	2023-07-25 20:35:35.026025800 +0200
@@ -124,7 +124,7 @@
     configure_file(${CMAKE_CURRENT_SOURCE_DIR}/platform/unix/BuildLinuxImage.sh.in ${CMAKE_CURRENT_BINARY_DIR}/BuildLinuxImage.sh @ONLY)
 endif ()
 
-target_link_libraries(OrcaSlicer libslic3r cereal)
+target_link_libraries(OrcaSlicer libslic3r cereal::cereal)
 if (APPLE)
 #    add_compile_options(-stdlib=libc++)
 #    add_definitions(-DBOOST_THREAD_DONT_USE_CHRONO -DBOOST_NO_CXX11_RVALUE_REFERENCES -DBOOST_THREAD_USES_MOVE)
diff -ruN ./src/libslic3r/CMakeLists.txt ../../source/src/libslic3r/CMakeLists.txt
--- ./src/libslic3r/CMakeLists.txt	1970-01-01 01:00:01.000000000 +0100
+++ ../../source/src/libslic3r/CMakeLists.txt	2023-07-25 20:36:03.784546604 +0200
@@ -480,7 +480,7 @@
 target_link_libraries(libslic3r
     libnest2d
     admesh
-    cereal
+    cereal::cereal
     libigl
     miniz
     boost_libs
diff -ruN ./src/slic3r/CMakeLists.txt ../../source/src/slic3r/CMakeLists.txt
--- ./src/slic3r/CMakeLists.txt	1970-01-01 01:00:01.000000000 +0100
+++ ../../source/src/slic3r/CMakeLists.txt	2023-07-25 20:36:21.337267677 +0200
@@ -496,7 +496,7 @@
 
 encoding_check(libslic3r_gui)
 
-target_link_libraries(libslic3r_gui libslic3r cereal imgui minilzo GLEW::GLEW OpenGL::GL hidapi ${wxWidgets_LIBRARIES} glfw libcurl OpenSSL::SSL OpenSSL::Crypto)
+target_link_libraries(libslic3r_gui libslic3r cereal::cereal imgui minilzo GLEW::GLEW OpenGL::GL hidapi ${wxWidgets_LIBRARIES} glfw libcurl OpenSSL::SSL OpenSSL::Crypto)
 #target_link_libraries(libslic3r_gui libslic3r cereal imgui minilzo GLEW::GLEW OpenGL::GL hidapi libcurl OpenSSL::SSL OpenSSL::Crypto ${wxWidgets_LIBRARIES} glfw)
 
 if (MSVC)
